src/components/MapEditorView/MapEditorView.module.scss

```diff
@@ -0,0 +1,19 @@
+.mapEditorView {
+  font-size: 16px;
+
+  .gridBorder {
+    overflow: hidden;
+    padding: 1em;
+    background: #fefefe;
+    border-right: 1px solid #d1d1d1;
+    border-bottom: 1px solid #d1d1d1;
+    border-left: 1px solid #d1d1d1;
+  }
+}
+
+/*
+  Class to use in Storybook to set editor width
+*/
+.mapEditorViewWidth {
+  max-width: 918px;
+}
```

Comment:
I usually set Storybook specific styles inline, to keep components free of Storybook stuff. The comment helps, but moving the code would be nice.

Reply:
Yes, will move it!